Frameworks/CoreGraphics/CGPath.mm

```diff
@@ -228,19 +233,20 @@ void CGPathAddLineToPoint(CGMutablePathRef path, const CGAffineTransform* m, flo
         return;
     }
 
-    if (m) {
-        CGPoint pt;
-
-        pt.x = x;
-        pt.y = y;
+    __preparePathForEditing(path);
 
+    CGPoint pt;
+    pt.x = x;
+    pt.y = y;
```

Comment:
nit: fold into `CGPoint pt{x, y};` to avoid using the assignment operator or copy constructors.

---
In reply to: [83128604](https://github.com/Microsoft/WinObjC/pull/1138#discussion_r83128604) [](ancestors = 83128604)